views/scss/inc/_test-layout.scss

```diff
@@ -91,6 +91,11 @@
             bottom: 0;
             width: 100%;
             opacity : .9;
+
+            &-full {
+                background-color: Window;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#System_Colors those values are deprecated. 
You can use `$uiGeneralContentBg`. Then it's the responsibility of the theme to override them.